test/test-generate-request-details.js

```diff
@@ -243,4 +244,26 @@ suite('Test Generate Request Details', function() {
     assert.equal(details.headers.Topic, extraOptions.headers.Topic);
     assert.equal(details.headers.Urgency, extraOptions.headers.Urgency);
   });
+
+  test('Audience contains port', function() {
+    const subscription = {
+      endpoint: 'http://example.com:4242/life-universe-and-everything'
+    };
+
+    const extraOptions = {
+      vapidDetails: {
+        subject: 'mailto:example@example.com',
+        publicKey: vapidKeys.publicKey,
+        privateKey: vapidKeys.privateKey
+      }
+    };
+
+    const message = undefined;
```

Comment:
Nit: remove this and simply pass `null` as the second argument to `generateRequestDetails`